packages/FieldService/src/Services/PartsConsumptionManager.php

```
@@ -50,17 +50,17 @@ public function recordConsumption(
         ]);
 
         // Get technician van warehouse ID
-        $vanWarehouseId = $this->getTechnicianVanWarehouseId($technicianId);
+        $vanWarehouseId = $this->warehouseManager->getTechnicianVanWarehouseId($technicianId);
 
         // Check van stock level
-        $vanStockLevel = $this->getStockLevel($vanWarehouseId, $productVariantId);
+        $vanStockLevel = $this->stockManager->getAvailableQuantity($vanWarehouseId, $productVariantId);
 
         $consumedFromVan = 0.0;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$consumedFromVan'. (undefined)

(UnusedLocalVariable)